docs/operations/create_index.mdx

```diff
@@ -10,7 +10,20 @@ description: A create index operation creates a new index on a set of columns.
   "create_index": {
     "table": "name of table on which to define the index",
     "name": "index name",
-    "columns": [ "names of columns on which to define the index" ]
+    "columns": [
+      "column_name": {
+        "collate": "collation name",
+        "sort": "ASC | DESC",
+        "nulls": "FIRST | LAST",
+        "opclass": {
+          "name": "operator_class_name",
+          "params": {
+             "param1": "val",
+             "param2": "val"
+          }
```

Comment:
I think this doc is out of date now that `params` was changed to a slice of strings rather than a map.